### PR TITLE
Adding the doc template to function oor

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Variables to be determined:
Inputs:      `r3polyToUse`, `STEP_SIZE`, `XGrid`, `YGrid`, `InterpolationF`
Outputs:   `improvedSkeleton`, `medIntensity`, `orthIntensVol`, `orthSamplePointsVol`, `skelScaleMm`

### CODE REVIEW

File: src/Functions/oor.m [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
-% Biomedicine, University of Luxembourg
-% 2016  - 2017
-% mail@andreashusch.de
+%    [improvedSkeleton, medIntensity, orthIntensVol, orthSamplePointsVol, skelScaleMm] 
+%    = oor(r3polyToUse, STEP_SIZE, XGrid, YGrid, interpolationF)
 %
-% That code needs some cleanup :-) But it works ;-)
-function [improvedSkeleton, medIntensity,orthIntensVol, orthSamplePointsVol, skelScaleMm] = oor(r3polyToUse, STEP_SIZE, XGrid, YGrid, interpolationF)
+% INPUTS: 
+%    r3polyToUse:
```
Comment (Daniel-Tojal) [inline src/Functions/oor.m]: @adhusch Could you please provide a description for these variables?

Comment (adhusch) [reply src/Functions/oor.m]: Inputs:
r3polyToUse: polynomial coefficient matrix
STEP_SIZE: step size along polynomial
Xgrid: grid to sample perpentidular to polyonial (x)
YGrid: grid to sample perpentidular to polyonial (y)
interpolationF: an interpolater fitted to point cloud (see scatteredInterpolant)

Comment (adhusch) [reply src/Functions/oor.m]: Outputs: 
improvedSkeleton: newly determined skeleton point cloud after re-sampling
medIntensity: median intensity along the polynomial
orthIntensVol: intensity volume stacked as orthogonal matrix
orthSamplePointsVols: sampling points stacked as orthogonal matrix
skelScaleMm: estimated milimetric scale

Comment (Daniel-Tojal) [reply src/Functions/oor.m]: Thank you!

File: src/Functions/oor.m [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
-% 2016  - 2017
-% mail@andreashusch.de
+%    [improvedSkeleton, medIntensity, orthIntensVol, orthSamplePointsVol, skelScaleMm] 
+%    = oor(r3polyToUse, STEP_SIZE, XGrid, YGrid, interpolationF)
 %
-% That code needs some cleanup :-) But it works ;-)
-function [improvedSkeleton, medIntensity,orthIntensVol, orthSamplePointsVol, skelScaleMm] = oor(r3polyToUse, STEP_SIZE, XGrid, YGrid, interpolationF)
+% INPUTS: 
+%    r3polyToUse:       Polynomial coefficient matrix
+%    STEP_SIZE:         Step size along polynomial
```
Comment (laurentheirendt) [inline src/Functions/oor.m]: May you align all the descriptions for inputs and outputs?

File: src/Functions/oor.m [hunk 3]
```diff
@@ -1,17 +1,29 @@
...
+%    interpolationF:    An interpolater fitted to point cloud (see scatteredInterpolant)
+%
+% OUTPUTS: 
+%    improvedSkeleton:          Newly determined skeleton point cloud after re-sampling
+%    medIntensity:              Median intensity along the polynomial
+%    orthIntensVol:             Intensity volume stacked as orthogonal matrix
+%    orthSamplePointsVol:       Sampling points stacked as orthogonal matrix
+%    skelScaleMm:               Estimated millimetric scale
+%
+% .. AUTHOR:
```
Comment (laurentheirendt) [inline src/Functions/oor.m]: You can actually put `AUTHORS` here.

**Review by laurentheirendt — CHANGES_REQUESTED** [review]

**Review by laurentheirendt — APPROVED** [review]

**Comment by laurentheirendt** [comment]
Thanks @Daniel-Tojal and @adhusch 👍

**Pull request merged**